implementations/micrometer-registry-statsd/src/test/java/io/micrometer/statsd/StatsdMeterRegistryTest.java

```diff
@@ -304,7 +302,7 @@ void counterIncrementDoesNotCauseStackOverflow() {
 
         // Cause the processor to get into a state that would make it perform logging at DEBUG level.
         ((Logger) LoggerFactory.getLogger(Operators.class)).setLevel(Level.DEBUG);
-        registry.processor.onComplete();
+        registry.manySink.tryEmitComplete();
```

Comment:
consider appending `.orThrow()`, so that unexpected failures to complete are detected (or comment very explicitly why the return value is ignored)